core/src/operations/mod.rs

```diff
@@ -392,6 +392,8 @@ pub enum Operation {
     /// the specified root will be removed from the advice provider. Otherwise, the advice
     /// provider will keep track of both, the old and the new advice sets.
     MrUpdate(bool),
+
+    Ext2Mul,
```

Comment:
nit: we should move it after field ops. And let's add doc comment which for now says just "TODO: add comments".